internal/gather/gather.go

```diff
@@ -76,6 +77,9 @@ func Data(clusterInfo *cluster.Info, status reporter.Interface, options Options)
 		options.Directory = "submariner-" + time.Now().UTC().Format("20060102150405") // submariner-YYYYMMDDHHMMSS
 	}
 
+	// concatenate the name of the cluster with the root gather directory
+	options.Directory = filepath.Join(options.Directory, clusterInfo.Name)
```

Comment:
Perhaps this should be optional in case your'e only gathering from a single cluster?

Otherwise the first directory will just have another nested in it, making it a bit useless.

Reply:
I think it’s worth keeping the cluster directory even if it’s the only directory, since the cluster name no longer appears in the generated file names. It also means the output is consistent regardless of how many clusters are “gathered”, and multiple “gather” directories for different clusters can easily be merged.